ext/groonga/rb-grn-object.c

```diff
@@ -1767,4 +1800,6 @@ rb_grn_init_object (VALUE mGrn)
                      rb_grn_object_accessor_p, 0);
     rb_define_method(rb_cGrnObject, "key_accessor?",
                      rb_grn_object_key_accessor_p, 0);
+
+    rb_define_method(rb_cGrnObject, "reindex", rb_grn_object_reindex, 0);
```

Comment:
I removed `reindex` from `Groonga::Object` and I added `Groonga::Database#reindex`.
Then, I'll add `Groonga::Table#reindex`.